shell/renderer/api/electron_api_context_bridge.cc

```diff
@@ -391,10 +412,10 @@ void ProxyFunctionWrapper(const v8::FunctionCallbackInfo<v8::Value>& info) {
     if (maybe_return_value.IsEmpty())
       return;
 
-    auto ret =
-        PassValueToOtherContext(func_owning_context, calling_context,
-                                maybe_return_value.ToLocalChecked(),
-                                &object_cache, support_dynamic_properties, 0);
+    auto ret = PassValueToOtherContext(
+        func_owning_context, calling_context,
+        maybe_return_value.ToLocalChecked(), &object_cache,
+        support_dynamic_properties, 0, BridgeErrorTarget::kDestination);
```

Comment:
Would it also be correct to use `TryCatch` here to capture the error and throw a new error in the correct context? e.g.
```suggestion
    v8::TryCatch try_catch(isolate);
    auto ret = PassValueToOtherContext(...);
    if (try_catch.HasCaught()) {
      ThrowException("Conversion of return value failed");
      return;
    }
```

Reply:
This is handled inside `PassValueToOtherContext`, the way that function is written and intended to be used is that it always does everything in the right context.  Any errors thrown are thrown in the correct context too so this try catch would be unnecessary.

Reply:
Yeah, I was just thinking about this as an alternative implementation that would be less intrusive (i.e. there would no need for this extra enum).

Reply:
It means we'd have to `trycatch` every call to `PassValueToOtherContext` which would be considerably more intrusive?